quixstreams/models/topics/manager.py

```diff
@@ -241,9 +241,9 @@ def topic_config(
     def topic(
         self,
         name: str,
-        value_deserializer: Optional[DeserializerType] = None,
+        value_deserializer: Optional[DeserializerType] = "json",
```

Comment:
I think I'm generally on board with this default change since it's what the app topic does anyway, but trying to remember if there was a reason to have not set it up that way.

Reply:
I couldn't figure out why. In any case it seems this is only called from the `Application.topic` method that uses this default.